consensus/abci.go

```diff
@@ -143,63 +175,32 @@ func (abci *TendermintABCI) CheckTx(txBytes []byte) tmtAbciTypes.ResponseCheckTx
 
 	from, err := ethTypes.Sender(signer, tx)
 	if err != nil {
-		abci.logger.Error(core.ErrInvalidSender.Error())
-		abci.metrics.CheckErrTxsTotal.Add(1, fmt.Sprint(abciTypes.ErrBaseInvalidSignature.Code))
-		return tmtAbciTypes.ResponseCheckTx{Code: uint32(abciTypes.ErrBaseInvalidSignature.Code), Log: core.ErrInvalidSender.Error()}
+		return common.Address{}, fmt.Errorf("INVALID_SENDER")
 	}
 
 	if tx.Value().Sign() < 0 {
-		abci.logger.Error(core.ErrNegativeValue.Error())
-		abci.metrics.CheckErrTxsTotal.Add(1, fmt.Sprint(abciTypes.ErrBaseInvalidInput.Code))
-		return tmtAbciTypes.ResponseCheckTx{Code: uint32(abciTypes.ErrBaseInvalidInput.Code), Log: core.ErrNegativeValue.Error()}
+		return common.Address{}, fmt.Errorf("INVALID_SIGNATURE")
```

Comment:
[MINOR] Encapsulate msg error into a constant somewhere else

Reply:
u can see where they have been used, only here so no constant needed for now it just makes it harder to maintain, jumping through codebase.